api-service/database/database.go

```diff
@@ -151,6 +151,12 @@ type MongoDatabaseInterface interface {
 	CountAlertsNODATA(alertsFilter dto.AlertsFilter) (int64, error)
 	// DismissHost dismiss the specified host
 	DismissHost(hostname string) error
+	// GetHostMinValidCreatedAtDate get the host's minimun valid CreatedAt date
+	GetHostMinValidCreatedAtDate(hostname string) (map[string]interface{}, error)
```

Comment:
Can we return a more clear value instead of `map[string]interface{}`?

Reply:
I agree with you! Now it returns time.Time.